src/pages/aurora-site/course-navigation.jsx

```
@@ -0,0 +1,557 @@
+import React, { useState, useRef, useEffect } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)